index.html

```diff
@@ -142,6 +142,18 @@ <h5>Languages</h5><img src="./images/Union.png" alt="expand" class="arrow"></li>
           <li class="expertise"><h5>Skills</h5><img src="./images/Union2.png" alt="expand" class="arrow"></li>
         </ul>
       </section>
+
+      <section id="form-section">
+        <h2>contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built ora project that needs coding. I’d love to help with it </p>
+
+        <form id="form" action="https://formspree.io/f/xjvlpwvy" method="post">
+          <input id="name" type="text" maxlength="30" value="Yeremias">
+          <input id="email" type="email" value="natajayanj14@gmail.com">
```

Comment:
- [ ] Please remove the `value` attribute for the input fields and include a `placeholder` text.
- [ ] Please include the required attribute to all fields to ensure the form is not submitted when any field is empty